queue-workers/handler.js

```diff
@@ -0,0 +1,48 @@
+const appInsights = require('applicationinsights');
+const logger = require('../utils/logger');
+const resUtil = require('../utils/res-util');
+const createTelegramBot = require('../utils/createTelegramBot');
+appInsights.setup(process.env.APPINSIGHTS_CONNECTIONSTRING).start();
+
+
+module.exports = async function (message, context) {
+    const { name, data } = message;
+    const { body, bot_token } = data;
+
+    
+    try {
+        const workerBot = createTelegramBot(bot_token);
+        const service = require(`../services/worker/${name}`);
+        const response = await service(body, workerBot);
+
+        if (response.status === 200) {
+            logger.info(response.data, `JOB-COMPLETED-${name.toUpperCase()}`);
```

Comment:
Do we have access to the job id from here? I think it is better use Job Id for the log key instead of name so it easier to trace logs